core/candleBatcher.js

```diff
@@ -6,22 +6,22 @@
 // Acts as ~fake~ stream: takes
 // 1m candles as input and emits
 // bigger candles.
-// 
+//
 // input are transported candles.
 
-var _ = require('lodash');
-var util = require(__dirname + '/util');
+const _ = require('lodash');
+const util = require(__dirname + '/util');
 
-var CandleBatcher = function(candleSize) {
-  if(!_.isNumber(candleSize))
+const CandleBatcher = function(candleSize) {
+  if (!_.isNumber(candleSize))
```

Comment:
this is not inline with any other if! All the ifs in the codebase have `if(` without a space!

Reply:
I see the problem, will change the styling for the if statements to be conform.

Reply:
thanks!

Reply:
According to the style guidelines from `AirBnB` and `Google`, as well as the `common` guideline, there should be a space between the `if` and the `(`.